src/client/eppo-client.ts

```diff
@@ -309,6 +339,30 @@ export default class EppoClient {
     );
   }
 
+  /**
+   * Initializes the `EppoClient` from the provided configuration. This method is async only to
+   * accommodate writing to a persistent store. For fastest initialization, (at the cost of persisting configuration),
+   * use `bootstrap` in conjunction with `MemoryOnlyConfigurationStore` instances which won't do an async write.
+   */
+  async bootstrap(configuration: IConfigurationWire): Promise<void> {
+    if (!configuration.config) {
+      throw new Error('Flag configuration not provided');
+    }
+    const flagConfigResponse: IUniversalFlagConfigResponse = inflateResponse(
+      configuration.config.response,
+    );
+    const banditParamResponse: IBanditParametersResponse | undefined = configuration.bandits
+      ? inflateResponse(configuration.bandits.response)
+      : undefined;
+
+    // We need to run this method sync, but, because the configuration stores potentially have an async write at the end
+    // of updating the configuration, the method to do so it also async. Use an IIFE to wrap the async call.
```

Comment:
🤨 
Why does this need to be run synchronously?

Reply:
It doesn't. I gave up trying to force `bootstrap` to be sync. The js-client `offlineInit` method is sync (in fact, the options type for that method is `IClientConfigSync`), so I need to understand a little more why that method is so explicitly sync.

We can also just fire and forget the async method to keep `bootstrap` sync, but that hanging promise feels awkward (and indeed eslint dislikes it)

Reply:
> We can also just fire and forget the async method to keep bootstrap sync, but that hanging promise feels awkward (and indeed eslint dislikes it)

I don't think that's an issue as nobody has a reason to await it, so letting it hang makes a lot of sense